roles/elastic-stack/ansible-kibana/tasks/main.yml

```diff
@@ -78,7 +79,7 @@
     dest: /etc/kibana/kibana.yml
     owner: root
     group: root
-    mode: '0664'
+    mode: 0664
```

Comment:
Kibana.yml can contain credentials, remove the readall permissions.

Please verify if it should be `group:kibana`

Reply:
![image](https://user-images.githubusercontent.com/14923294/76885885-3e630200-6880-11ea-8dab-aaa533d2bda9.png)

Reply:
Default kibana permissions and owner:group after fresh repo install